lib/html_to_plain_text.rb

```diff
@@ -147,5 +149,9 @@ def format_list_item(out, options)
         out << "#{number}. "
       end
     end
+
+    def data_table?(table)
```

Comment:
now changed to `table.attributes['border'].to_s.to_i > 0`

On Fri, Dec 4, 2015 at 10:56 AM, Brian Durand <notifications@github.com>
wrote:

> In lib/html_to_plain_text.rb
> <https://github.com/bdurand/html_to_plain_text/pull/5#discussion_r46717554>
> :
>
> > @@ -147,5 +149,9 @@ def format_list_item(out, options)
> >          out << "#{number}. "
> >        end
> >      end
> > +
> > +    def data_table?(table)
>
> This method seems mis-named since I would assume "data" tables would be
> the ones with borders.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bdurand/html_to_plain_text/pull/5/files#r46717554>.
>